app/views/sitemap/index.xml.haml

```diff
@@ -7,12 +12,27 @@
       %changefreq monthly
       %priority 0.5
 
-    %url
-      %loc= url_for controller: 'pages', action: 'badge', host: 'localhost', trailing_slash: true
-      %changefreq monthly
-      %priority 0.5
+  %url
+    %loc #{pages_question_url}/
+    %changefreq monthly
+    %priority 0.5
 
-    %url
-      %loc= url_for controller: 'pages', action: 'question', host: 'localhost', trailing_slash: true
-      %changefreq monthly
-      %priority 0.5
\ No newline at end of file
+  %url
+    %loc #{pages_badge_url}/
+    %changefreq monthly
+    %priority 0.5
+  
+  %url
+    %loc #{new_user_session_url}/
+    %changefreq monthly
+    %priority 0.5
+  
+  %url
+    %loc #{new_user_password_url}/
+    %changefreq monthly
+    %priority 0.5
+
+  %url
+    %loc #{new_user_confirmation_url}/
+    %changefreq monthly
+    %priority 0.5
```

Comment:
well sitemaps need to include urls that contain most of the content.i am not sure about it either.then i may remove the password and confirmation url.but i think keeping the sign in  page is better.